docs/src/data/meta.ts

```diff
@@ -10,31 +14,36 @@ export const META_INFO: MetaInfo = {
       metaTitle: 'Home',
       metaDescription:
         'Amplify UI is an open-source design system with cloud-connected components and primitives that simplify building accessible, performant, and beautiful applications in React, Angular, Vue, and Flutter (more coming soon).',
+      supportedFrameworks: 'react|angular|vue|flutter',
     },
   },
   '/[platform]': {
     frontmatter: {
       metaTitle: 'Home',
       metaDescription:
         'Amplify UI is an open-source design system with cloud-connected components and primitives that simplify building accessible, performant, and beautiful applications in React, Angular, Vue, and Flutter (more coming soon).',
+      supportedFrameworks: 'react|angular|vue|flutter',
     },
   },
   '/404': {
     frontmatter: {
       metaTitle: '404',
       metaDescription: 'Page Not Found',
+      supportedFrameworks: 'react|angular|vue|flutter',
```

Comment:
Should we have an `all` flag so we don't have to list all the frameworks? Also, if supportedFrameworks is missing, we should assume all frameworks are supported. We don't want 404's to stop working just because we forgot to add a new framework (say React Native).